### DISCUSSION TITLE
Hides leaderboard in multiplayer / Lazer

### DISCUSSION THREAD

**Discussion by Cefaxion** [ppy/osu#27615] [Ideas]
Able to hide or unhide the leaderboard, i play on android and the leaderboard always hides some circles, changin the UI Size isnt a Fix for that, if i play on full screen size my thumbs and hand hurts a lot for the extra space that I have to hold the cursor
Playing like this always the leaderboards hides the circles
![Screenshot_2024-03-14-21-24-35-746_sh ppy osulazer](https://github.com/ppy/osu/assets/121911998/73483b5c-9436-4f84-8ca9-40f0c23401ca)

**Comment by Gabixel** [comment]
I think the reason can be found [here](https://github.com/ppy/osu/discussions/22212#discussioncomment-4688761).
There are already discussions about this in general (not specific to Android), but I guess it's still a no-go for this feature.
- https://github.com/ppy/osu/discussions/26024
- https://github.com/ppy/osu/discussions/17747
- https://github.com/ppy/osu/discussions/26473

(Maybe being able to _scale_ the leaderboard could be something to consider?)

**Reply by Sinaeb** [reply]
everything during gameplay should be hideable, anything that cannot be hidden is something that clutters just becomes a large messy blur that distracts from the gameplay
